.github/workflows/prt_labels.yml

```diff
@@ -35,7 +35,7 @@ jobs:
                 repo: context.repo.repo,
                 issue_number: prNumber,
               });
-              const labelsToRemove = ['PRT-Failed', 'PRT-Passed'];
```

Comment:
Could you elaborate on ` labels are removed and the PR looks all green even if the PRT failed earlier.`?
And, shouldn't we remove both of thses labels if new commit is added/amend'ed?

Reply:
If some check is failed on the PR, it shows as crossed on the PR but when all the PRT labels are removed then it shows the PR as green in the Pull Requests list. With this change, we are trying to keep the PR in failed state unless PRT is passed on it.

Reply:
Yes, the change makes sense, and it shows PR as green with other checks but not PRT which isn't run yet, and we check PRT results only.
In earlier approach we kept no PRT results labels after new commit added, and now we keep PRT failed/passed until we run PRT after new commit is added (iiuc). Although I'd prefer the earlier approach, I am fine with either one, and as @omkarkhatavkar already ack'ed, so I wouldn't block this.